turtlebot3c-gadget/gadget/gadget-amd64.yaml

```diff
@@ -0,0 +1,62 @@
+volumes:
```

Comment:
why is this file duplicated in `turtlebot3c-gadget/gadget` and in `turtlebot3c-gadget/`?